Aleksandr Zorin/index.html

```diff
@@ -0,0 +1,60 @@
+<!doctype html>
+<html>
+<head>
+<meta charset="utf-8">
+<title>Tokyo</title>
+<link rel="stylesheet" type="text/css" href="css/style.css" />
+<link href="https://fonts.googleapis.com/css?family=Dosis:200,300,400&display=swap" rel="stylesheet">
+</head>
+
+<body>
+	<article class="weatherWidget">
+		<div class="location">
+			<div class="weatherTypeLogo"><img src="img/weather/sunny-big.png" width="50%"></div>
+			<div class="weatherType">Sunny</div>
+			<div class="today">Tokyo - Wednesday, July 31</div>
+		</div>
+		<div class="information">
+			<div class="temp">
+				28 &deg;C
+				<hr>
+			</div>
+			<div class="graph">
+				<svg height="80%" width="100%">
+ 					<line x1="20" y1="10" x2="80" y2="130" stroke="#8E4140" stroke-width = "2"/>
+ 					<line x1="80" y1="130" x2="140" y2="70" stroke="#8E4140" stroke-width = "2"/>
+ 					<line x1="140" y1="70" x2="200" y2="30" stroke="#8E4140" stroke-width = "2"/>
+ 					<line x1="200" y1="30" x2="260" y2="130" stroke="#8E4140" stroke-width = "2"/>
+ 					<line x1="260" y1="130" x2="320" y2="10" stroke="#8E4140" stroke-width = "2"/>
+ 					<line x1="320" y1="10" x2="380" y2="70" stroke="#8E4140" stroke-width = "2"/>
+		
+  					<circle cx="20" cy="10" r="4" stroke="#8E4140" stroke-width="3" fill ="white"/>
+  					<circle cx="80" cy="130" r="4" stroke="#8E4140" stroke-width="3" fill ="white"/>
+  					<circle cx="140" cy="70" r="4" stroke="#8E4140" stroke-width="3" fill ="white"/>
+  					<circle cx="200" cy="30" r="4" stroke="#8E4140" stroke-width="3" fill ="white"/>
+  					<circle cx="260" cy="130" r="4" stroke="#8E4140" stroke-width="3" fill ="white"/>
+  					<circle cx="320" cy="10" r="4" stroke="#8E4140" stroke-width="3" fill ="white"/>
+  					<circle cx="380" cy="70" r="4" stroke="#8E4140" stroke-width="3" fill ="white"/>
+  				</svg>
+				<hr>
+			</div>
+			<div class="days">
+				<div class="day">
+					<div class="wrapper"><img src="img/weather/sunny.png" width="70%"></div>
```

Comment:
@Zorin-Alex Name of class usually should represent aim of the tag. Try to avoid of generic class names. So 'wrapper' here can be more specific and be named as 'weatherImgWrapper' or something like this